utils/abi-coder.js

```diff
@@ -971,6 +971,19 @@ utils.defineProperty(Coder.prototype, 'encode', function(names, types, values) {
     return utils.hexlify(coderTuple(this.coerceFunc, coders).encode(values));
 });
 
+utils.defineProperty(Coder.prototype, 'decodeRevertReason', function(data) {
+
+    data = utils.arrayify(data);
+
+    if (data && utils.hexlify(data.slice(0, 4)) === '0x08c379a0') {
+        var result = coderTuple(this.coerceFunc, [
+            getParamCoder(this.coerceFunc, {type: 'string'}, undefined),
```

Comment:
If you do not need the last param, do not pass it as opposed to passing undefined